config/locales/services/payments/fr.yml

```diff
@@ -0,0 +1,6 @@
+---
+fr:
+  services:
+    payments:
+      errors:
+        select_payment_method: "Choisissez une méthode de paiement"
```

Comment:
it's quite possible this request lacks the correct tone (I dropped "please" from the message). My initial gut feeling was _"choisissez une méthode de paiement, s.v.p"_ or _"vous devez choisir une methode de paiement"_ and it might make sense to leave this in English rather than worry about it more.

The reason I put it in french at all was the validate the translation was happening correctly:

```ruby
[1] pry(main)> I18n.locale=:fr
=> :fr                        
[2] pry(main)> Payments::ProcessCreditPurchase.call(:nobody, :noamount, purchase_options: {})
=> #<Payments::ProcessCreditPurchase:0x000055ce864dcf60                                      
 @credits_count=:noamount,
 @error="Choisissez une méthode de paiement",
 @purchase_options={},
 @success=false,
 @user=:nobody>
[3] pry(main)> I18n.locale=:en
=> :en                        
[4] pry(main)> Payments::ProcessCreditPurchase.call(:nobody, :noamount, purchase_options: {})
=> #<Payments::ProcessCreditPurchase:0x000055ce8658cc58                                      
 @credits_count=:noamount,
 @error="Please select a payment method",
 @purchase_options={},
 @success=false,
 @user=:nobody>
```

I'll add a unit test for this, asserting when no payment option is selected a translated error is returned.